### PR TITLE
Merge latest changes from `harjaapdhillon16/i-am-human-dapp` repository

### PR DESCRIPTION
**Pull request by harjaapdhillon16**
- Added tests using Jest for manageTokens
- Added github workflow (with eslint, prettier formatter)

### CODE REVIEW

**Comment by robert-zaremba** [comment]
1. Please make the title more meaningful - this title doesn't make sense
2. In description, you should write the summary of the updates.
3. This PR is too big, combine too many things, and it is hard to review. Why all of it goes in single Pull Request? Megha tests should come separately, otherwise, by default, everything will be merged as a single commit.

**Comment by robert-zaremba** [comment]
4. Auto lint should definitely go into a separate PR (like changing " -> ')
5. There are many removed lines (+3k vs -24k). Could you summarize what was remove?

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      - name: Build packages
+        env:
+          CI: false
+        run: yarn build
+      - name: Run test
+        run: yarn test
+      - name: Execute format
+        run: yarn format
+      - name: Execute lint
+        run: yarn lint
```
Comment (robert-zaremba) [inline .github/workflows/main.yml:51]: lint and format should go into a separate job to enable parallel execution

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "semi": true,
+    "singleQuote": true
```
Comment (robert-zaremba) [inline .prettierrc:5]: if we remove this line, will we have smaller diff?

**Comment by robert-zaremba** [comment]
also, names of the commits are not very meaningful . First 23 commits are name "changes" or "fixes"

File: .github/workflows/main.yml (hunk 1 above)
Comment (Megha-Dev-19) [reply .github/workflows/main.yml:51]: okay will make the update

File: .prettierrc (hunk 2 above)
Comment (Megha-Dev-19) [reply .prettierrc:5]: yes it will reduce the changes, do you want to me add the prettier config in a new PR

**Comment by Megha-Dev-19** [comment]
> 4. Auto lint should definitely go into a separate PR (like changing " -> ')

will keep that in mind in future
> 5. There are many removed lines (+3k vs -24k). Could you summarize what was remove?

the major changes is from formatter like you said changing " -> ', then since we added more packages , yarn.lock file changes are also there. Apart from this, I have added tests and github workflow (mentioned in description)

File: .prettierrc (hunk 2 above)
Comment (robert-zaremba) [reply .prettierrc:5]: Yes, let's remove this line

File: .github/workflows/main.yml (hunk 1 above)
Comment (robert-zaremba) [reply .github/workflows/main.yml:51]: this could go in a separate PR.

**Comment by robert-zaremba** [comment]
Had a chat with @Megha-Dev-19 . Since most of the work here is related to @Megha-Dev-19 changes, we should review her work. In that case, let's close this PR, and @Megha-Dev-19 will create a new one.
